lib/internal/crypto/keys.js

```diff
@@ -634,7 +637,7 @@ function isKeyObject(obj) {
 // here similar to other things like URL. A chromium provided CryptoKey
 // will not be recognized as a Node.js CryptoKey, and vice versa. It
 // would be fantastic if we could find a way of making those interop.
-class CryptoKey extends JSTransferable {
```

Comment:
This is a breaking change. While WebCrypto doesn't define CryptoKey as transferable, we've implemented it this way from the beginning. Especially since we've already marked Web Crypto as stable we should preserve this.

Reply:
Would you take a closer look please? The change does not seem to have impacted its transferrability.

Reply:
cc @addaleax @jasnell

Reply:
ping @addaleax @jasnell